harness/determined/common/api/request.py

```diff
@@ -356,3 +363,37 @@ def ws(host: str, path: str) -> WebSocket:
     token = authentication.must_cli_auth().get_session_token()
     websocket.add_header("Authorization".encode(), "Bearer {}".format(token).encode())
     return WebSocket(websocket)
+
+
+def parse_warnings(
```

Comment:
Pretty sure this function is not necessary.

Pretty sure `bindings.v1LaunchWarning(x)` is what you are looking for; it should work with whatever the REST API emits.

This is a snippet from the generated bindings for how the conversion is done there:

```python
[v1LaunchWarning(x) for x in obj["warnings"]] if obj["warnings"] is not None else None
```